src/Quartz.Tests.Integration/Quartz.Tests.Integration.csproj

```diff
@@ -70,6 +70,26 @@
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="FakeItEasy, Version=1.25.2.0, Culture=neutral, PublicKeyToken=eff28e2146d5fd2c, processorArchitecture=MSIL">
```

Comment:
Do we still need hard references like these? I was so pleased about the project.json resolving issue by being the glue between csproj and packages.